qiita_pet/webserver.py

```diff
@@ -57,6 +57,16 @@
     ReloadPluginAPItestHandler)
 from qiita_pet import uimodules
 from qiita_db.util import get_mountpoint
+from qiita_pet.handlers.rest import (
+    StudyHandler as WeDontKnowWhatToCallThis,
```

Comment:
We've put all these under a constant, this avoids the namespace collisions :) we think this is better.